docs/source/index.rst

```diff
@@ -51,3 +51,24 @@ See the sections below (and the top-level links above) for more information abou
    extension/extension_dev
    developer/contributing
    privacy_policies
+
+.. .. only:: html
+
+   <div class="alert alert-warning" role="alert">
+      <strong>Note:</strong> The following section provides guidelines for menu item labeling based on Apple's Human Interface Guidelines.
+   </div>
```

Comment:
I am not sure if this should be an alert. It currently does not render at all:

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/83da651a-12ed-4754-b7f6-1468e6a43274)

If it should be a visible admonition, let's implement it using the native admonitions syntax, for example with `.. note:`

https://github.com/jupyterlab/jupyterlab/blob/73a947de96941d7d254f397d7f2cbb0df7f69958/docs/source/user/toc.rst?plain=1#L24-L31

which render like this:

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/b9b229af-880f-4fc4-aff8-7e8d5684b302)

See https://jupyterlab.readthedocs.io/en/latest/user/toc.html